_overviews/scala3-book/ca-extension-methods.md

```diff
@@ -50,6 +50,25 @@ Then in your code you use `circumference` just as though it was originally defin
 aCircle.circumference
 ```
 
+### Import extension method
+
+Imagine, that `circumference` is defined in package `lib`, you can import it by
+
+```scala
```

Comment:
IIRC we can’t set the language name in front of the code fences. Did you check that it works well?